modules/aws-eks-managed-node-groups/locals.tf

```diff
@@ -66,11 +66,7 @@ locals {
     templatefile("${path.module}/templates/userdata-${local.managed_node_group["launch_template_os"]}.tpl", local.userdata_params)
   )
 
-  eks_worker_policies = toset(concat([
-    "${local.policy_arn_prefix}/AmazonEKSWorkerNodePolicy",
-    "${local.policy_arn_prefix}/AmazonEKS_CNI_Policy",
-    "${local.policy_arn_prefix}/AmazonEC2ContainerRegistryReadOnly",
-    "${local.policy_arn_prefix}/AmazonSSMManagedInstanceCore"],
+  eks_worker_policies = toset(concat(
```

Comment:
we probably can remove eks_worker_policies and directly for_each from `local.managed_node_group["additional_iam_policies"]` , but wanted to get your opinion on this

Reply:
Code looks cleaner this way. We can keep it like that